media-libs/mesa/mesa-13.0.5.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=5
 
-EGIT_REPO_URI="git://anongit.freedesktop.org/mesa/mesa"
+EGIT_REPO_URI="https://anongit.freedesktop.org/${PN}/${PN}"
```

Comment:
Oh, the correct URL is actually https://anongit.freedesktop.org/git/mesa/mesa.git

I'll update it and check all other ebuilds to ensure none of them have "https://anongit.freedesktop.org/" followed by something other than "git/"